File: Iniciante/1060 - Números Positivos/Python/1060.py
def main():
    lista = entradas()
    print(f"{positivo(lista)} valores positivos")


def entradas():
    lista = []
    for entrada in range(6):
        valor = float(input())
        lista.append(valor)
    return lista


def positivo(l):
    positivos = 0
    for item in l:
        if item >= 0:
            positivos += 1
    return positivos


main()

File: Iniciante/1060 - Números Positivos/TypeScript/1060.ts
import * as fs from "fs";

function main(): void {
  const input: string = fs.readFileSync("/dev/stdin", "utf8");
  const lines: string[] = input.split("\n");
  let positiveValues: number = 0;
  for (let i = 0; i < 6; i++) {
    let value: number = parseFloat(lines.shift()!);
    if (value > 0) {
      positiveValues++;
    }
  }
  console.log(`${positiveValues} valores positivos`);
}

main();
